code/src/components/Card.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+// import '../index.css'
+
+export const Card = ({ props }) => {
+  return (
+    <section className="album-card">
+      <container className="images-container">
+        <div className="icons">
+          <img
+            className="heart-icon"
+            src="./icons/heart.svg"
+            alt="like button" />
+          <img
+            className="play-icon"
+            src="./icons/play.svg"
+            alt="play button" />
+          <img
+            className="dots-icon"
+            src="./icons/dots.svg"
+            alt="more button" />
+        </div>
+        <img
+          src={props.images[0].url}
+          alt={props.name}
+          className="album-img" />
```

Comment:
Perhaps add an _href_ here so that when you click on the album cover you get re-directed to the album on Spotify.